kernel_handlers.py

```diff
@@ -125,22 +152,21 @@ def getresuid_entry_handler(syscall_id, syscall_object, pid):
     ruid_addr = cint.peek_register(pid, cint.EBX)
     euid_addr = cint.peek_register(pid, cint.ECX)
     suid_addr = cint.peek_register(pid, cint.EDX)
-
+    
     logging.debug('ruid: %d', ruid)
     logging.debug('euid: %d', euid)
     logging.debug('suid: %d', suid)
-
+    
     logging.debug('ruid addr: %x', ruid_addr & 0xffffffff)
     logging.debug('ruid addr: %x', euid_addr & 0xffffffff)
     logging.debug('ruid addr: %x', suid_addr & 0xffffffff)
     noop_current_syscall(pid)
-
+    
     cint.populate_unsigned_int(pid, ruid_addr, ruid)
     cint.populate_unsigned_int(pid, euid_addr, euid)
     cint.populate_unsigned_int(pid, suid_addr, suid)
     apply_return_conditions(pid, syscall_object)
 
-
```

Comment:
Gotta leave this blank line here, two blank lines to separate methods by python standards